backend/tarsy/integrations/mcp/client.py

```
@@ -47,13 +47,22 @@
+                logger.debug(f"  Command: {server_config.connection_params.get('command')}")
+                logger.debug(f"  Args: {server_config.connection_params.get('args', [])}")
+                logger.debug(f"  Env: {server_config.connection_params.get('env', None)}")
+                
                 # Create server parameters for stdio connection
                 server_params = StdioServerParameters(
                     command=server_config.connection_params.get("command"),
                     args=server_config.connection_params.get("args", []),
                     env=server_config.connection_params.get("env", None)
                 )
                 
+                logger.debug(f"Created StdioServerParameters for '{server_id}': command='{server_params.command}', args={server_params.args}, env={server_params.env}")
```

Ruff (0.12.2)

Line too long (167 > 88)

(E501)